Numbers.java

```diff
@@ -0,0 +1,109 @@
+package Lesson_12;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class Numbers {
+    ArrayList<Integer> numbersList = new ArrayList<>();
+    char choice;
+    boolean isPrime=true;
+
+    public Numbers (){
+        this.numbersList=numbersList;
+        this.choice=choice;
+        this.isPrime=isPrime;
+    }
+
+
+    public ArrayList addNumbersToList(char inputChoice) {
+        Scanner reader = new Scanner(System.in);
+        Integer addNumber = 0;
+        if (inputChoice == 'y') {
+            do {
+                System.out.println("Type any number:");
+                addNumber = reader.nextInt();
+                this.numbersList.add(addNumber);
+                System.out.println("Add another number? y/n");
+                this.choice = reader.next().toLowerCase().charAt(0);
+            } while (this.choice == 'y');
+        }else System.out.println("Ok. Thank you.");
+        return this.numbersList;
+    }
+
+    public void printList (){
+        System.out.println(this.numbersList);
+    }
+
+    public int sumOfNumbersInList(){
+        int count = 0;
+        int sum = 0;
+        while (this.numbersList.size() > count) {
```

Comment:
Since count is increasing, it would be more understandable if you define this while condition like:
```
while (count < this.numbersList.size()) {
 ...
}
```

Reply:
Also, easier approach is to use just for loop like: 
```
for (int number : this.numbersList) {
  sum += number;
}
```